await-wrap.js

```diff
@@ -149,3 +149,12 @@ class AwaitWrap {
 }
 
 module.exports = AwaitWrap;
+
+/**
+ * @function isAsyncFn
+ * @param {Function} fn Function to check type of
+ * @returns {Boolean} true/false
+ */
+function isAsyncFn(fn) {
+  return toString.call(fn).toLowerCase().slice(8, -1) === 'asyncfunction';
```

Comment:
:trollface: 

```js
async function(data, done) {
  const result = await something(data);
  done(result);
}
```

Not that we should ever write that or need to support such an abomination here, but 🤣

Reply:
:trollface: